skaffold.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: golang-proxy
 build:
   artifacts:
-    - image: lingo
+    - image: us-central1-docker.pkg.dev/substratus-dev/substratus/lingo
```

Comment:
Any reason why you use GAR here? We have lingo published for free to ghcr.

Reply:
Not really, I wanted to be able to use skaffold from my local machine (I guess that doesnt matter here) and also didnt want to need to rely on other github actions running. Any thoughts?

Reply:
In that case you can revert the change to the image and it should all just work

Reply:
NVM I misunderstood how this worked outside of kind.